tests/test_unit/test_connections/test_copy_connection.py

```diff
@@ -100,8 +98,6 @@ async def test_maintainer_plus_can_copy_connection_with_delete_source(
         "user": "user",
         "password": "password",
     }
-    assert origin.is_deleted == is_delete_source
```

Comment:
If `is_delete_source=True`, original connection should be deleted, and we should check for this.

Reply:
for connections I left `is_deleted` field as there is a huge logic for copying deleted connections between the groups